Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/CredentialFeatureID.swift

```diff
@@ -0,0 +1,32 @@
+//
```

Comment:
Namespace enum for credential feature IDs relevant to Swift SDK.

e, f, j, l, m, x, y are skipped either because they're not part of SEPs for resolvers we implemented or because they don't apply to Swift infrastructure.